Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM rust:1-slim-stretch as builder
+
+WORKDIR /srv/addresses-importer
+
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get install -y gcc-4.8 && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+ENV CC "gcc-4.8"
+RUN git clone https://github.com/openvenues/libpostal \
+    && cd postal \
+    && ./bootstrap.sh \
+    && ./configure --datadir=$(pwd)/data \
+    && make install \
+    && cd .. \
+    && ldconfig
+
+COPY . ./
+
+RUN cd deduplicator && cargo build --release
+
+FROM debian:stretch-slim
+
+WORKDIR /srv
+
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get install -y gcc-4.8 && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+COPY --from=builder /srv/addresses-importer/deduplicator/target/release/deduplicator /usr/bin/deduplicator
```

Comment:
If I understand well, we are now exporting necessary files in a newly created docker image. Won't we miss a part of dependencies here? At least libpostal and its metadata and some of its dependencies?